src/main/java/cal/internshipmanager/controller/InternshipApplicationController.java

```diff
@@ -1,6 +1,8 @@
 package cal.internshipmanager.controller;
 
```

Comment:
Add default status when creating application

Reply:
I added the status pending approval when its created